### ISSUE TITLE
"Proper" order and format for RetroArch launch paths

### ISSUE THREAD

**Issue by drinfernoo** [zach-morris/plugin.program.iagl#64]
It may be a non-issue, but I ran across this while working in my aarch64 PR... but while referring to [the documentation for RetroArch](https://github.com/libretro/RetroArch/blob/master/pkg/android/phoenix-gradle/README.md), I noticed a few things.

* The extra parameters in the launch path are appended with the `--es` flag

* The names *and* the parameters themselves are in quotes

* The order of the parameters *seems* to matter

I couldn't, for example, get the RetroArch (aarch64) paths to work unless I had them in the following order:

`/system/bin/am start --user 0 -a %ACTION% -c %CATEGORY% -n %ACTIVITY% --es "LIBRETRO" "%COREPATH%" --es "CONFIGFILE" "%CFG_PATH%" --es "ROM" "%ROM_PATH%" --es "IME" "%IME%"`

For the action, category, activity and IME parameters, they are basically the same (hence me not expressing them fully here), but the activity parameter needed to specify *which* app it was coming from.

Basically, the difference was that the activity parameter had to come after the category parameter, and the ROM parameter had to come *after* the core and config parameters.

This issue is simply to wonder if it would be prudent to make the same adjustments to the 32bit RetroArch launch paths, in order to maintain some consistency between them. Obviously, the paths for the 32bit version seem to be working fine, so this may be mostly cosmetic, but may also improve compatibility into the future, if any breaking changes are made to RetroArch.
